node/src/Commands.ts

```diff
@@ -1526,6 +1526,20 @@ export function createBLPop(
     return createCommand(RequestType.BLPop, args);
 }
 
+/**
+ * @internal
+ */
+export function createFunctionLoad(
+    libraryCode: string,
+    replace?: boolean,
+): command_request.Command {
+    const args =
+        replace === undefined || !replace
+            ? [libraryCode]
+            : ["REPLACE", libraryCode];
```

Comment:
Is it possible to simplify to 
```suggestion
    const args =
        replace ?
            ["REPLACE", libraryCode]
            : [libraryCode];
```